cpp/include/cudf/table/row_operators.cuh

```diff
@@ -463,11 +463,12 @@ class row_hasher {
  public:
   row_hasher() = delete;
   row_hasher(table_device_view t) : _table{t} {}
+  row_hasher(table_device_view t, uint32_t seed) : _table{t}, _seed(seed) {}
 
   __device__ auto operator()(size_type row_index) const
   {
-    auto hash_combiner = [](hash_value_type lhs, hash_value_type rhs) {
-      return hash_function<hash_value_type>{}.hash_combine(lhs, rhs);
+    auto hash_combiner = [=](hash_value_type lhs, hash_value_type rhs) {
+      return hash_function<hash_value_type>{_seed}.hash_combine(lhs, rhs);
```

Comment:
The seed isn't relevant to `hash_combine`, yes? 
```suggestion
      return hash_function<hash_value_type>{}.hash_combine(lhs, rhs);
```

Reply:
You're right. Just pushed a fix.